src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -403,6 +421,24 @@
             az acr build-task run -n MyBuildTask -r MyRegistry
 """
 
+helps['acr build-task update-build'] = """
+    type: command
+    short-summary: Patch the build properties.
+    examples:
+        - name: Update an existing build to be archived or not.
```

Comment:
what does archiving a build mean? it also seems weird to make it a part of build-task, since they should be independent things, but i trust you guys have thought out all the weird cli/rp hierarchy stuff

Reply:
I didn't totally agree, but the spirit was to put everything (except quick build) under build task.